util/paused/paused.go

```diff
@@ -32,10 +32,22 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	clusterv1beta1 "sigs.k8s.io/cluster-api/api/core/v1beta1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/core/v1beta2"
 	"sigs.k8s.io/cluster-api/util/annotations"
-	"sigs.k8s.io/cluster-api/util/conditions"
-	"sigs.k8s.io/cluster-api/util/patch"
+	"sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions"
+	patch "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/patch"
```

Comment:
```suggestion
	v1beta1patch "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/patch"
```

Let's be explicit and also add this to golangci-lint config :-)

Reply:
Fixed and added alias to the golangci-lint config, thanks!